.github/workflows/inferno-test.yml

```diff
@@ -0,0 +1,67 @@
+name: Inferno Certification Test
+
+on:
+    push:
+        branches: [ master ]
+    pull_request:
+        branches: [ master ]
+
+permissions:
+    contents: read
+
+jobs:
+    inferno_certification_test:
+        runs-on: ubuntu-22.04
```

Comment:
hi @adunsulag, was looking at some of the other workflows and was wondering if it would help for this to be `ubuntu-latest`?

Reply:
It works.  Thanks for the suggestion.